client/dashboard/store-performance/index.js

```diff
@@ -201,15 +200,11 @@ export default compose(
 		const userIndicators = indicators.filter(
 			( indicator ) => ! hiddenBlocks.includes( indicator.stat )
 		);
-		const { woocommerce_default_date_range: defaultDateRange } = select(
-			SETTINGS_STORE_NAME
-		).getSetting( 'wc_admin', 'wcAdminSettings' );
 
 		const data = {
 			hiddenBlocks,
 			userIndicators,
 			indicators,
-			defaultDateRange,
```

Comment:
Removed the `defaultDateRange ` since this can be retrieved from the parent component `client/dashboard/customizable.js `.